app/src/main/java/org/thoughtcrime/securesms/calls/WebRtcCallActivity.kt

```diff
@@ -104,9 +107,11 @@ class WebRtcCallActivity : PassphraseRequiredActionBarActivity() {
 
         // Only enable auto-rotate if system auto-rotate is enabled
         if (isAutoRotateOn()) {
```

Comment:
This can change while the activity is created or started... maybe even resumed IDK. Do we want to handle it?

Reply:
The only way to do this without messing with configuration changes and custom layouts is to always listen for the sensor and bail early if auto rotate is off, which is now checked on every sensor event.
I don't think it's a costly call so it should be fine.